client/components/FormattingBar/media/MediaTwitter.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react';
+import { InputGroup, Button, Intent, NonIdealState } from '@blueprintjs/core';
+import { isHttpsUri } from 'valid-url';
+
+import Icon from 'components/Icon/Icon';
+import { apiFetch } from 'client/utils/apiFetch';
+import { getIframeSrc, getEmbedType } from 'client/utils/editor';
+
+type Props = {
+	onInsert: (...args: any[]) => any;
+};
```

Comment:
`onInsert` expects an argument that contains the type of node to insert. I didn't see a type definition for that yet anywhere, so I just made it a string. But if there is an enum for it you can point me to that I'm missing because I'm new to it, lmk.